os/kernel/binary_manager/binary_manager.h

```diff
@@ -68,9 +68,16 @@ enum loading_thread_cmd {
 	LOADCMD_LOAD = 0,
 	LOADCMD_LOAD_ALL = 1,
 	LOADCMD_RELOAD = 2,
+	LOADCMD_UPDATE = 3,
 	LOADCMD_LOAD_MAX,
 };
 
+enum reload_type_e {
+	RELOAD_FAULT = 0,
+	RELOAD_UPDATE = 1,
```

Comment:
RELOAD is kind of verbose, it means that put the same thing to somewhere once again.
UPDATE also kind of verbose, it means that change new one.
VERBOSE + VERBOSE is not good option for variable or enum.
I guess you want classify reload's behavior with these enums, then rename is needed.
(e.g RELOAD_TYPE_OLD /* With same item */, RELOAD_TYPE_NEW /* with New Item */ ....)

Reply:
Thanks. I will more thinks.